ox-builder.gemspec

```diff
@@ -27,6 +27,6 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'benchmark-ips'
   gem.add_development_dependency 'builder'
 
-  gem.add_dependency 'ox', ['~> 2.3.0']
-  gem.add_dependency 'docile', ['~> 1.1.5']
+  gem.add_dependency 'ox', ['~> 2.8.0']
+  gem.add_dependency 'docile', ['~> 1.3.0]
```

Comment:
You're missing a closing quote here.

Reply:
Also, I'm not sure I want to exclude older versions of `ox` and `docile`, do you have a reason for this? Using `>=` instead of `~>` should work.

Reply:
> You're missing a closing quote here.

This'll teach me to rely on the GitHub web UI for editing :) Fixed!

I tend to let my users just use an old version of my libraries when they need an old version of the dependencies. But there doesn't seem to be any breakage when using old versions here. So I've changed both these lines to a `>=`.

Thanks for this, it'll allow my team to update to a recent Ruby release!